source/_data/projects.yml

```diff
@@ -0,0 +1,33 @@
+# open source projects that use Cypress to write end-to-end tests
+
+- name: Redash
+  description: "Connect and query your data sources, build dashboards to visualize data and share them with your company."
+  url: https://github.com/getredash/redash/tree/master/client
+
+- name: Mermaid.js
+  description: "Generation of diagram and flowchart from text in a similar manner as markdown"
+  url: https://github.com/mermaid-js/mermaid
+
+- name: WithSpectrum
+  description: "Simple, powerful online communities at spectrum.chat"
+  url: https://github.com/withspectrum/spectrum
+
+- name: Vue Storefront
+  description: "PWA for eCommerce that is 100% offline, platform agnostic, and headless"
+  url: https://github.com/DivanteLtd/vue-storefront/tree/master/test/e2e
+
+- name: Netlify CMS
+  description: "A CMS for Static Site Generators at netlifycms.org"
+  url: https://github.com/netlify/netlify-cms/tree/master/cypress
+
+- name: ngx-bootstrap
+  description: Fast and reliable Bootstrap widgets in Angular
+  url: https://github.com/valor-software/ngx-bootstrap/tree/development/cypress
+
+- name: "octokit/rest.js"
+  description: GitHub REST API client for JavaScript
+  url: https://github.com/octokit/rest.js/tree/master/test
+
+- name: For All Beautiful Earth
+  description: End-to-end browser tests for fab.earth using Cypress.io test runner
+  url: https://github.com/forallabeautifulearth/fabe-browser-tests
```

Comment:
These links are inconsistent. They vary between:

- Link to main GitHub repo
- Link to sub folder in GitHub repo
- Link to exact cypress folder with tests in GitHub repo. 
- Some of them link to a specific tree

If any of these projects just change their file structure, these links will break. I suggest linking to the projects main GitHub repo - not any branch or subfolder.

Reply:
I was trying to point at the close "e2e" folder, because some projects are large and use different folder names, but yeah it makese more sense to just point at the repo